### PR TITLE
[OYPD-147] Blog page full + teaser view mode

### PR DESCRIPTION
**Pull request by hamrant**
Old PR - https://github.com/ymcatwincities/openy/pull/70

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build255

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/255/

File: modules/openy_features/openy_node/modules/openy_node_blog/config/install/field.field.node.blog.field_blog_location.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ id: node.blog.field_blog_location
 field_name: field_blog_location
 entity_type: node
 bundle: blog
-label: Location
+label: Locations
```
Comment (alexschedrov) [inline modules/openy_features/openy_node/modules/openy_node_blog/config/install/field.field.node.blog.field_blog_location.yml:13]: Should be update in documentation as well.

File: modules/openy_features/openy_node/modules/openy_node_blog/config/install/field.field.node.blog.field_blog_related.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ id: node.blog.field_blog_related
 field_name: field_blog_related
 entity_type: node
 bundle: blog
-label: 'Related Blog posts'
+label: 'Related content'
```
Comment (alexschedrov) [inline modules/openy_features/openy_node/modules/openy_node_blog/config/install/field.field.node.blog.field_blog_related.yml:11]: Should be update in documentation as well.

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build255/profiles/openy/build/reports/behat

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build258

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/258/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build258/profiles/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**